src/main/kotlin/nl/info/client/brp/util/BRPClientHeadersFactory.kt

```diff
@@ -37,7 +34,7 @@ class BRPClientHeadersFactory @Inject constructor(
     companion object {
         private const val X_API_KEY = "X-API-KEY"
         private const val X_ORIGIN_OIN = "X-ORIGIN-OIN"
-        private const val X_DOELBINDING = "X-DOELBINDING"
+        const val X_DOELBINDING = "X-DOELBINDING"
```

Comment:
is this still the right location for these constants? Maybe they should be pulled out of the object.